src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveSummaryService.kt

```diff
@@ -159,6 +162,20 @@ data class IepResult(
   val daysOnLevel: Int
 )
 
+fun daysOnLevel(iepDetails: List<IepDetail>): Int {
```

Comment:
can this `daysOnLevel` utility function be private? Or do the tests need this to be public to use it?

Reply:
i tried that but the tests couldn't see it even though they're in the same package. not really sure what private even means when it's top-level in a package.